### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by brianorwhatever**

### CODE REVIEW

**Review by dlongley — CHANGES_REQUESTED** [review]
`package-lock.json` needs to be removed.

File: tests/20-main.spec.js [hunk 1]
```diff
@@ -261,7 +261,7 @@ describe('statusTypeMatches', () => {
     err.message.should.contain('"credentialStatus" is invalid');
   });
 
-  it('should not match when "CONTEXTS.VC_BRL_V1" is not in ' +
+  it('should not match when "CONTEXTS.VC_BSL_V1" is not in ' +
```
Comment (dlongley) [inline tests/20-main.spec.js]: This is true for 1.x VCs ... if we don't have a test for 2.x VCs, we should add one such that if the VC 2.0 context is present, then the BSL context needn't be (in order for the status to match).

File: tests/mock-sl-credentials.js [hunk 2]
```diff
@@ -19,9 +19,9 @@ export const slCredentialRevocation = {
     SUITE_CONTEXT_URL
   ],
   id: 'https://example.com/status/1',
-  issuer: 'did:key:z6MkesAkkxuETfHCMdv3gRTKr6iFiQZMjGT5pM8745oSzGX1',
+  issuer: 'did:key:z6MkowtnRyMkCerXvXqjCYUo2mLEeCgX1sWfoP2CZA5UjBop',
   issuanceDate: '2022-06-02T16:00:21Z',
-  type: ['VerifiableCredential', 'BitstringStatusListCredential'],
+  type: [ 'VerifiableCredential', 'BitstringStatusListCredential' ],
```
Comment (dlongley) [inline tests/mock-sl-credentials.js]: This is an incorrect style change:

```suggestion
  type: ['VerifiableCredential', 'BitstringStatusListCredential'],
```

File: tests/20-main.spec.js [hunk 3]
```diff
@@ -1183,7 +1237,7 @@ describe('assertBitstringStatusListContext', () => {
     err.message.should.contain('first "@context" value');
   });
 
-  it('should fail when "CONTEXTS.VC_BRL_V1" is not in "@context"', async () => {
+  it('should fail when "CONTEXTS.VC_BSL_V1" is not in "@context"', async () => {
```
Comment (dlongley) [inline tests/20-main.spec.js]: Same situation for VC 2.0 context here.

**Review by dlongley — APPROVED** [review]
Approving as this looks like it fixes previously broken tests (I presume), however, there are a couple of suggestions here that could be added either in this PR or in a follow on one. Also see the minor style nit. Thanks!

File: tests/20-main.spec.js (hunk 1 above)
Comment (brianorwhatever) [reply tests/20-main.spec.js]: I have reworded this test to include "or VC 2". We will need to add new tests for all of the VC 2 work but this rounds out he VC 1 work I believe

**Pull request merged**